### PR TITLE
Offline server

### PR DESCRIPTION
**Pull request by BartMoroz**

### CODE REVIEW

File: applications/offline-server/main.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+	db, _ = scribble.New(".", nil)
+	port := ":8080"
+	router := mux.NewRouter().StrictSlash(true)
+	router.PathPrefix("/js/").Handler(http.StripPrefix("/js/", http.FileServer(http.Dir(dir))))
+	router.HandleFunc("/v1/tokens", Tokens)
+	router.HandleFunc("/v1/orders", Orders)
+	router.HandleFunc("/v1/transactions", Transactions)
+	router.HandleFunc("/", HomePage)
+	http.Handle("/", router)
+	fmt.Println("Serving worldpay web service mock on port " + port)
```
Comment (maciej-wakula-mobica) [inline applications/offline-server/main.go]: Would be better to use formatters here

File: applications/offline-server/main.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+	if err := db.Write(DB_NAME, orderRequest.CustomerOrderCode, orderInformation); err != nil {
+		fmt.Println("Error", err)
+	}
+	json.NewEncoder(w).Encode(t)
+}
+
+func Transactions(w http.ResponseWriter, r *http.Request) {
+	records, err := db.ReadAll(DB_NAME)
+	if err != nil {
+		fmt.Println("ERRRRORRUUU: ", err)
```
Comment (maciej-wakula-mobica) [inline applications/offline-server/main.go]: Errrrrrrorruuuuu?????
Muuuuuuuuu.... secret cow level...

File: applications/offline-server/static/.idea/static.iml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (maciej-wakula-mobica) [inline applications/offline-server/static/.idea/static.iml]: remove & add gitignore

File: applications/offline-server/static/js/app.js [hunk 4]
```diff
@@ -0,0 +1 @@
+$(document).foundation()
```
Comment (maciej-wakula-mobica) [inline applications/offline-server/static/js/app.js]: not needed?

File: applications/offline-server/static/index.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+	<meta http-equiv="x-ua-compatible" content="ie=edge"></meta>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0"></meta>
+	<title>The Smart Parking Demo - transactions</title>
+	<script src="js/jquery-2.1.4.js"></script>
+</head>
+
+<body class="mainBgColorRed fullBodyFont">
+	<div class="tb" id="tabelka">
+	</div>
+	<script src="js/wp-demo-prod.js"></script>
```
Comment (maciej-wakula-mobica) [inline applications/offline-server/static/index.html]: is the JS filename correct ?!?

File: applications/offline-server/main.go (hunk 2 above)
Comment (BartMoroz) [reply applications/offline-server/main.go]: fmt.Println("DB readAll error: ", err)

**Pull request merged**